examples/platforms/nrf52811/README.md

```diff
@@ -0,0 +1,266 @@
+# OpenThread on nRF52811 Example
+
+This SoC is meant to be used in the configuration that involves the Host Processor and the IEEE 802.15.4 radio.
+In this configuration, the full OpenThread stack is running on the Host Processor and the nRF52811 SoC acts as an IEEE 802.15.4 radio.
+The radio is running a minimal OpenThread implementation that allows for communication between the Host Processor and the nRF52811.
+
```

Comment:
Can we mention that for full OpenThread on SoC please take a look on nRF52840 platform.

With a link to nRF52840?

Reply:
fixed